neqo-transport/src/connection/tests/idle.rs

```diff
@@ -709,12 +706,6 @@ fn keep_alive_with_ack_eliciting_packet_lost() {
     let retransmit = client.process_output(now).dgram();
     assert!(retransmit.is_some());
 
-    // The next callback should be for an idle PING.
-    assert_eq!(
-        client.process_output(now).callback(),
-        IDLE_TIMEOUT / 2 - pto
-    );
-
     // Wait that long and the client should send a PING frame.
```

Comment:
```suggestion
    // Wait for half the idle timeout (less the PTO we've already waited)
    // so that we get a keep-alive. The actual timeout is the smaller of this
    // duration or PTO*2, whichever is smaller.
```

Reply:
Done. Thanks.